b9lab-content/4-running-a-chain/01-index.md

```diff
@@ -1 +1,3 @@
 # Running a Chain Overview
+
+In this section, you clone the `cosmos-sdk` repo first. Then you start and test an educational sample. By following these steps, you have access to the `cosmos-sdk` codebase and the Cosmos SDK components so you don't have to generate new code. 
```

Comment:
@CitMC Should it be "In this module"? `4-running-a-chain` is a module, right?

Reply:
@xavierlepretre Yes, it should be either module or chapter.